src/hotspot/share/utilities/vmError.cpp

```diff
@@ -1167,6 +1167,10 @@ void VMError::report(outputStream* st, bool _verbose) {
       st->cr();
     }
 
+  STEP_IF("printing fast locked objects", _verbose && _thread != nullptr && _thread->is_Java_thread() && LockingMode == LM_LIGHTWEIGHT);
+    st->print_cr("Objects fast locked by this thread (top to bottom):");
+    JavaThread::cast(_thread)->lock_stack().print_on(st);
+
```

Comment:
I would have expected this to be printed along with the other current thread info.

"fast locked" is not terminology we are using for this any more IIUC. I would suggest just saying this is the lock stack for the thread.

Reply:
I had put it at the end of the "T H R E A D" section. It's moved a bit up, now. Changed wording and added newline.